handler/src/main/java/io/netty/handler/stream/ChunkedStream.java

```diff
@@ -99,8 +100,14 @@ public void close() throws Exception {
         in.close();
     }
 
+    @Deprecated
```

Comment:
@normanmaurer - I think what I have here is what we want ... I just did a quick test to verify that is concrete types also do not have the `@Deprecated` annotation on the method, a compiler warning will be generated for that class indicating a deprecation warning.

IFoo.java
```java
public interface IFoo {
    @Deprecated
    void foo();
}
```

MyFoo.java
```java
public final class MyFoo implements IFoo {
    @Deprecated
    @Override
    public void foo() { // Line 5 if @Deprecated is removed
        System.err.println("Deprecated");
    }
}
```

FooTest.java
```java
public class FooTest {
    public static void main(String[] args) {
        IFoo ifoo = new MyFoo();
        MyFoo myFoo = new MyFoo();

//        ifoo.foo();
        myFoo.foo(); // Line 23
    }
}
```

If I removed `@Deprecated` I get a warning on the `MyFoo.java` file:
```
MyFoo.java:5: warning: [deprecation] foo() in IFoo has been deprecated
    public void foo() {
```

With the `@Deprecated` the deprecated warning is then only on usages of that method:
```
FooTest.java:23: warning: [deprecation] foo() in MyFoo has been deprecated
        myFoo.foo();
```